### ISSUE TITLE
CA2252 false positives in VS Code after OmniSharp update

### ISSUE THREAD

**Issue by alexrp** [OmniSharp/omnisharp-roslyn#2404]
I have a solution that sets `LangVersion=preview` and `EnablePreviewFeatures=true`. I've confirmed that the compiled assembly gets `[RequiresPreviewFeatures]` as expected. Yet, I get `CA2252` in the editor on code like this:

```csharp
namespace Vezel.Novadrop.Data.Serialization.Items;

interface IDataCenterItem<T>
    where T : unmanaged
{
    static abstract void ReverseEndianness(ref T item);
}
```

This didn't happen until I updated OmniSharp today.

Can be repro'd by cloning and building https://github.com/vezel-dev/novadrop at `0df3de26a6f65ff9ad12c1df7a25772a964efc84` and opening it up in VS Code.

Using C# extension `1.25.0-win32-x64` and OmniSharp `1.38.3-beta.87-net6.0`.

**Comment by filipw** [comment]
why are you using 1.25.0 with an old version of OmniSharp? 1.25.0 bundles OmniSharp 1.39.0. Make sure to clear any `omnisharp.path` setting you might have

**Comment by alexrp** [comment]
@filipw `omnisharp.path` is/was `latest`. Checking today, I'm now on `1.39.1-beta.2-net6.0` (I think it updated yesterday or so).

**Comment by alexrp** [comment]
From what I can tell, this issue doesn't exist anymore, so closing.
